commons/ui/src/main/res/values-ro-rRO/strings.xml

```diff
@@ -7,4 +7,5 @@
     <string name="title_discover">Descoperă</string>
     <string name="title_themes">Categorii de lecții</string>
     <string name="min">min</string>
+    <string name="cancel">RENUNȚĂ</string> <!--niciodată-->
```

Comment:
I don't get the comment?

Reply:
I meant it as a joke. We should never give up.